Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -250,14 +250,16 @@ try {
                         if ($depth -gt 1) {
                             # When there are multiple build jobs, we need to add build job specific project list (and count) to the output of the Initialization job
                             $initializationOutputs = $yaml.Get('jobs:/Initialization:/outputs:/')
-                            $addOutput = @()
-                            1..$depth | ForEach-Object {
-                                $addOutput += @(
-                                  "projects$($_): `${{ steps.BuildOrder.outputs.projects$($_)Json }}"
-                                  "projects$($_)Count: `${{ steps.BuildOrder.outputs.projects$($_)Count }}"
-                                )
+                            if ($initializationOutputs) {
```

Comment:
If there is no output, shouldn't the process create it?

Reply:
No, this is when downgrading - the old PR handler doesn't have the Outputs section.

Reply:
But the same problem exists with the build part, downgrading a repo now to test this